src/main/resources/reference.conf

```diff
@@ -46,7 +46,8 @@ jdbc-journal {
   parallelism = 8
 
   slick {
-    driver = "slick.jdbc.PostgresProfile$"
+    profile = "slick.jdbc.PostgresProfile$"
+    profile = ${?jdbc-journal.slick.driver}
```

Comment:
This will make sure that an user that have declared the `driver`, but not the `profile` will have his driver config used as a `profile`